packages/contracts/contracts/minter-suite/Minters/MinterSetPriceERC20V4.sol

```diff
@@ -361,6 +443,25 @@ contract MinterSetPriceERC20V4 is
     function purchaseTo_do6(
         address _to,
         uint256 _projectId
```

Comment:
would be fine removing this gas-optimized version in this update

Reply:
Updated!